scripts/splitClasses.test.js

```diff
@@ -0,0 +1,13 @@
+const splitClasses = require('./splitClasses').splitClasses
+
+test('Returns a list of list of classes', () => {
+  const oldLog = console.log
+  console.log = jest.fn()
```

Comment:
A probably better way is to mock console.log, like in https://github.com/cozy/cozy-client/blob/90c7abb13fd71267cb44a1af6bca7b03b03a7645/packages/cozy-client/src/queries/dsl.spec.js#L6

```
jest.spyOn(console, 'log').mockImplementation(() => jest.fn())
...
expect(console.log).toHaveBeenCalledWith(...)
```